src/gluonts/nursery/autogluon_tabular.py

```diff
@@ -117,7 +113,46 @@ def train(self, training_data: Dataset) -> TabularPredictor:
             for entry in training_data
         ]
         df = pd.concat(dfs)
-        ag_model = self.task.fit(df, label="target")
+
+        if "excluded_model_types" not in self.kwargs:
+            excluded_model_types = ["KNN", "XT", "RF"]
+        else:
+            excluded_model_types = self.kwargs["excluded_model_types"]
+
+        if "hyperparameters" not in self.kwargs:
+            hyperparameters = "light"
+        else:
+            hyperparameters = self.kwargs["hyperparameters"]
+
+        if "num_bagging_sets" not in self.kwargs:
+            num_bagging_sets = 1
+        else:
+            num_bagging_sets = self.kwargs["num_bagging_sets"]
+
+        if "presets" not in self.kwargs:
+            presets = [
+                "best_quality_with_high_quality_refit",
+                "optimize_for_deployment",
+            ]
+        else:
+            presets = self.kwargs["presets"]
+
+        if "eval_metric" not in self.kwargs:
+            eval_metric = "mean_absolute_error"
+        else:
+            eval_metric = self.kwargs["eval_metric"]
+
```

Comment:
I think you can do these in a more manageable way as follows: within `__init__` do

```python
default_kwargs = {
    "excluded_model_types": ["KNN", "XT", "RF"],
    "hyperparameters": "light",
    ...
}
self.kwargs = {**default_kwargs, **kwargs}
```

this way, for any key which is in both `kwargs` and `default_kwargs`, the value in `kwargs` will take precedence